File: Java/src/secao05/Exercicio05.java
package secao08;

import java.util.Scanner;

public class Exercicio05 {
    public static void main(String[] args) {
        // var
        int[] vetor = new int[10];
        boolean tem_maior_50 = false;
        Scanner teclado = new Scanner(System.in);

        // input/processing
        for (int i = 0; i < vetor.length; i++) {
            System.out.printf("Informe um número para o vetor (%d/10): ", (i + 1));
            vetor[i] = teclado.nextInt();
        }
        for (int i = 0; i < vetor.length; i++) {
            if (vetor[i] > 50) {
                System.out.printf("O número %d (maior que 50) está na posição %d.\n", vetor[i], i);
                tem_maior_50 = true;
            }
        }
        if (!tem_maior_50) {
            System.out.println("Não existe nenhum número maior que 50.");
        }
        teclado.close();
    }
}

File: Python/secao05/Exercicio05.py
# var
vetor = []
posicao = 0
tem_maior_50 = False

# input
for i in range(0, 10):
    num = int(input("Informe um valor para o vetor"))
    vetor.append(num)

for i in vetor:
    if i > 50:
        print("O número {0} está na posição {1} do vetor.".format(i, posicao))
        tem_maior_50 = True
    posicao = posicao + 1
if tem_maior_50 == False:
    print("Não existe nenhum número maior que 50")